### PR TITLE
Moving more startup logic into the BigtableSession startup.

### PR DESCRIPTION
**Pull request by sduskis**
I think the deferred startup was causing the performance tests that were blocking the launch of 0.2.0

### CODE REVIEW

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java [hunk 1]
```diff
@@ -102,6 +103,13 @@
   private static final Logger LOG = new Logger(BigtableSession.class);
 //  private static final SslContextBuilder sslBuilder = createGrpcSslBuilder();
```
Comment (kevinsi4508) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: remove

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java [hunk 2]
```diff
@@ -102,6 +103,13 @@
...
+  private static ExecutorService connectionStartupExecutor =
+      Executors.newCachedThreadPool(
+          new ThreadFactoryBuilder()
+              .setNameFormat("BigtableSession-startup-%s")
+              .setDaemon(true)
+              .build());
+
   static {
     performWarmup();
   }
```
Comment (kevinsi4508) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: Remove below comment if it is not needed anymore.

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java (hunk 1 above)
Comment (sduskis) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: This is not related to the current PR.  We can address it elsewhere.

File: bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java (hunk 2 above)
Comment (sduskis) [inline bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java]: This is not related to the current PR.  We can address it elsewhere.

**Comment by kevinsi4508** [comment]
LGTM
